non-regression/non-regression.sh

```diff
@@ -18,9 +18,10 @@ echo "[ with Tableaux-CDCL ]"
 echo "===============================================================================================";
 ./main_script.sh "-sat-solver Tableaux-CDCL"
 
-echo "===============================================================================================\c";
+echo "===============================================================================================";
 echo "===============================================================================================";
 echo "[ WITH FM-SIMPLEX ]"
-echo "===============================================================================================\c";
 echo "===============================================================================================";
-./main_script.sh "-inequalities-plugin `pwd`/../sources/fm-simplex-plugin.cmxs"
+echo "===============================================================================================";
+./main_script.sh "-inequalities-plugin `pwd`/../sources/_build/install/default/share/plugins/fm-simplex-plugin.cmxs"
```

Comment:
Can we have/need a link for plugins in sources/ ?

Reply:
Are plugins sufficiently frequently used to need symlinks ?